src/Raven.Server/Documents/Commands/Replication/GetReplicationOutgoingsFailureInfoCommand.cs

```diff
@@ -29,6 +34,37 @@ public override HttpRequestMessage CreateRequest(JsonOperationContext ctx, Serve
             return request;
         }
 
+        public override async Task<ResponseDisposeHandling> ProcessResponse(JsonOperationContext context, HttpCache cache, HttpResponseMessage response, string url)
```

Comment:
why do you need this?

Reply:
otherwise, we get an exception (where the object's format does not match the expected one, resulting in an `EndOfStreamException`). once we address https://issues.hibernatingrhinos.com/issue/RavenDB-22177/Error-in-DynamicJsonValue.ConvertTK-TV-JsonDeserializationBase.ToDictionaryTK-TV-when-Deserializing-Non-Primitive-Types-as, we can modify `ReplicationOutgoingsFailurePreview.ToJson()` and likely eliminate the need for `ProcessResponse()`.